### PR TITLE
FEAT: Animated Background Gradient Image

### PR DESCRIPTION
**Pull request by Flowdeeps**
First pass at having a gradient image produced by the browser

### CODE REVIEW

**Comment by Flowdeeps** [comment]
Parked for ease of packaging - Do Not Merge

File: routes/editor.js [hunk 1]
```diff
@@ -432,6 +476,44 @@ router.post('/page/imagebackground/id/:id', function (req, res) {
...
             srcmedium: newItem['srcmedium']
         };
+        var gradientEnable = (newItem['gradientEnable'] ? true : false);
+        var gradientAnimate = (newItem['gradientAnimate']) ? true : false;
+        var gradientDirection = newItem['gradientDirection'];
+        var gradientSpeed = newItem['speed'];
+        var gradientStops = [];
+        console.log(newItem['stop']);
+        for (var stop in newItem['stop']) {
+          if (newItem['stop'][stop] !== '') {
```
Comment (naomiaro) [inline routes/editor.js]: this is where the problem would happen with string vs array splitting to letters

File: views/editor/fragments/gradientstop.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class="js-Stop js-Dynamic">
+  <input type="text" placeholder="&hellip;" value="" name="stop" />
```
Comment (naomiaro) [inline views/editor/fragments/gradientstop.html]: `name="stop" ` is singular here. Try using `name="stops[]" `

File: views/editor/fragments/gradientstops.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<fieldset>
+  <legend>Stops</legend>
+  <div>
+    <label>Colour</label>
+    <div class="js-ContentDynamic">
+    {{#gradient}}
+    {{#stops}}
+      <div class="js-Stop js-Dynamic">
+        <input type="text" placeholder="&hellip;" name="stop" value="{{.}}" />
```
Comment (naomiaro) [inline views/editor/fragments/gradientstops.html]: this again should use the array form

**Pull request merged**